app/experimenter/nimbus-ui/src/lib/mocks.tsx

```diff
@@ -421,9 +421,10 @@ fiveDaysAgo.setDate(fiveDaysAgo.getDate() - 5);
 /** Creates a single mock experiment suitable for getAllExperiments queries.  */
 export function mockSingleDirectoryExperiment(
   overrides: Partial<getAllExperiments_experiments> = {},
+  slugIndex: number = Math.round(Math.random() * 100),
 ): getAllExperiments_experiments {
   return {
-    slug: `some-experiment-${Math.round(Math.random() * 100)}`,
+    slug: `some-experiment-${slugIndex}`,
```

Comment:
This is what fixes the issue because we call this function [here](https://github.com/mozilla/experimenter/blob/2497b9c205c76d14360ba90fde59b31e3d50f11d/app/experimenter/nimbus-ui/src/lib/mocks.tsx#L492) and it receives each object `override` + index.